### ISSUE TITLE
Gradebook displays all due dates and total points regardless of the filter

### ISSUE THREAD

**Issue by cigas** [RunestoneInteractive/RunestoneServer#2004]
This seems like new behavior in the past week

**What Course are you in**
CS152HOFA2022

**What Page were you on**
Gradebook

**What is your username**
cigas

**Describe the bug**
It looks like the due dates were added to the top rows of the gradebook display in the past few days, but these don't get adjusted when applying a filter. I see now the same applies to total points. See images.

![day4-2](https://user-images.githubusercontent.com/11619455/187324583-6d312b3b-8ed1-4e87-a2b7-a881f2b815ab.PNG)

![day4-3](https://user-images.githubusercontent.com/11619455/187324589-22041dbe-4028-46d3-8111-6f84b0d754a6.PNG)

**Comment by bnmnetp** [comment]
Hmmm I haven't touched any gradebook code in a very long time...

Not sure exactly what you are seeing there.

**Comment by cigas** [comment]
The due date line doesn't change - it shows all the due dates for the entire term. It should be filtered to only show the due dates for the specified columns. Same for total points. If you look at the line class average, it does change, showing only the day 04 assignments/columns.

**Comment by cigas** [comment]
Yes, I am still seeing this behavior. Here's perhaps a simpler example.
No filtering
![no-filter](https://user-images.githubusercontent.com/11619455/211865077-bd4c0fc9-027d-4622-bda0-69113dbdb9b7.PNG)
Filter - Due date and Total Points are not filtered, so remain as the previous first column.
![filter](https://user-images.githubusercontent.com/11619455/211865392-81faee92-4426-47eb-9c51-a79e4eb3d4be.PNG)

**Comment by Tanaquil18** [comment]
It's still like this though.  The Date and Total Points don't change when you start typing what assignment you want to Filter By.
